packages/react/src/components/DateTimePicker/DateTimePicker.test.jsx

```diff
@@ -229,10 +229,6 @@ describe('DateTimePicker', () => {
     expect(wrapper.find(`.${iotPrefix}--date-time-picker__field`).first().text()).toEqual(
       '2020-04-01 13:34 to 2020-04-06 11:49'
     );
-
-    wrapper.find(`.${iotPrefix}--date-time-picker__menu-btn-apply`).first().simulate('click');
-    jest.runAllTimers();
-    expect(dateTimePickerProps.onApply).toHaveBeenCalled();
```

Comment:
Why was this removed? The portion above is clicking the increment button setting a valid date and time range, so apply should be enabled and the callback fired.

Reply:
Re-added the test to reflect changed made.